pyglmnet/pyglmnet.py

```diff
@@ -123,20 +124,6 @@ def __init__(self, distr='poisson', alpha=0.05,
         self.eta = eta
         set_log_level(verbose)
 
-    def __repr__(self):
```

Comment:
`BaseEstimator` has it already (https://github.com/scikit-learn/scikit-learn/blob/51a765a/sklearn/base.py#L274), you like this representation better?